cellprofiler/modules/_help.py

```diff
@@ -220,29 +217,10 @@
 The resulting image can also be saved with the **SaveImages** module.
 """
 
-####################
-#
-# ICONS
-#
-####################
-def __image_resource(filename):
-    #If you're rendering in the GUI, relative paths are fine
-    if os.path.relpath(pkg_resources.resource_filename(
-        "cellprofiler",
-        os.path.join("data", "images", filename)
-    )) == os.path.join("cellprofiler","data", "images", filename):
-        return os.path.relpath(pkg_resources.resource_filename(
-            "cellprofiler",
-            os.path.join("data", "images", filename)
-        ))
 
-    # Otherwise, we are probably building the documentation in sphinx.
-    # The path separator used by sphinx is "/" on all platforms.
-    return "../images/{}".format(filename)
+# TODO: Move these too??
```

Comment:
Where would a better place for this be? It seems to be a common help resource, so it makes sense to me to leave it here.

Reply:
These images are only used in the context of modules, so I decided to leave these variables here. I will remove the TODO comment.